x-pack/plugins/index_management/public/sections/index_list/components/index_actions_context_menu/index_actions_context_menu.container.js

```diff
@@ -30,18 +31,23 @@ import {
 
 import {
   getIndexStatusByIndexName,
-  getIndicesByName
+  getIndicesByName,
+  getKbnIndex,
 } from '../../../../store/selectors';
 
 const mapStateToProps = (state, ownProps) => {
   const indexStatusByName = {};
   const { indexNames } = ownProps;
+  const kbnIndexName = chrome.getInjected('kbnIndex');
+
   indexNames.forEach((indexName) => {
     indexStatusByName[indexName] = getIndexStatusByIndexName(state, indexName);
   });
+
   return {
     indexStatusByName,
-    indices: getIndicesByName(state, indexNames)
+    indices: getIndicesByName(state, indexNames),
+    kbnIndex: getKbnIndex(state, indexNames, kbnIndexName)
```

Comment:
Instead of looking for the Kibana index specifically, I think we should look for any index that begins with a period, since that's how we identify system indices. This might capture some indices which can safely be deleted (Kibana indices leftover from prior versions, for example), but it's guaranteed to capture all indices which _can't_ safely be deleted, so I think it's better to be safe than sorry.

![image](https://user-images.githubusercontent.com/1238659/59621259-c6df2700-90e3-11e9-9268-9a01cc2705c4.png)

Reply:
good suggestion! i'll work on making this change.